source/2019-09-27-the-ember-times-issue-117.md

```diff
@@ -26,80 +32,96 @@ responsive: true
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Accessibility, Compiler Secrets and Modern Ember @ EmberCamp 🐹](#section-url)
+
+
+Last week the Ember community went on another extraordinary trip into the metropolitan wilderness of Illinois:
+[**EmberCamp 2019**](https://embercamp.com) took place in Chicago on September 16th and was insightful, exciting and entertaining as always!
+
+With twelve, versed speakers, the schedule offered thoughtful insights into:
+
+- what's new in Ember with its [first edition, Octane](https://emberjs.com/editions/octane/)
+- how [Embroider](https://github.com/embroider-build/embroider) and Ember will literally build a future together
+- everything you need to know to build **accessible JavaScript apps** in 2019
+- how to level up your app's **performance** with worker super powers or well-balanced animations
```

Comment:
I think animations may be a counterpoint to improving performance. To list both workers and animations under the same item, maybe:
```suggestion
- how to level up your app with *web workers** and meaningful **animations**
```